core/src/main/java/org/springframework/security/core/GrantedAuthority.java

```diff
@@ -31,6 +31,8 @@
  */
 public interface GrantedAuthority extends Serializable {
 
+	String MISSING_AUTHORITIES_ATTRIBUTE = GrantedAuthority.class + ".missingAuthorities";
```

Comment:
What is this intended to be used for? Thus far it seems that it is the `HttpServletRequest` attribute name. If it is only used for web (I think that is the only place we will use it), I believe this makes more sense in the web jar / package, perhaps in `WebAttributes`.